### DISCUSSION TITLE
Search Backoffice Pages By Title

### DISCUSSION THREAD

**Discussion by GioAc96** [mia-platform/community#340] [Ideas]
### Which area of this project could be most improved?

Backoffice

### Description

Hi! On our backoffice, we have a huge number of pages (> 150) split into several categories (> 25). As you can imagine, finding a specific page is quite time consuming, especially if you don't remember exactly what the page name is. Would it be possible to add a search feature for backoffice pages?

It would also be nice if each page also had one or more alternative names that it could be matched by (e.g. the "Tickets" page could also match the "travel title" or "reservation" query strings)

**Comment by TheRusCompass** [comment]
Hi @GioAc96 thanks for the proposal! We can surely add a search bar! For what concerns the alternative names we will think about a possible solution, but it should be feasible.

**Comment by LiliumCandidum** [comment]
Hi @GioAc96 , we released the back-kit version 1.5.0 with the feature you asked for.
Using the component `bk-layout`, you can set the new property `showSearchBar` to `true` to display the search bar. It looks for every item in the menu based on `label` and `tags`. `tags` is a new property of the menu items and it's an array of strings.

**Reply by GioAc96** [reply]
Awesome! Thank you, we'll check it out
